Networking/NetworkingTests/Mapper/WordPressThemeListMapperTests.swift

```diff
@@ -0,0 +1,40 @@
+import XCTest
+@testable import Networking
+
+final class WordPressThemeListMapperTests: XCTestCase {
+
+    /// Verifies that the whole list is parsed.
+    ///
+    func test_WordPressThemeListMapper_parses_all_contents_in_response() throws {
+        let themes = try mapLoadWordPressThemeListResponse()
+        XCTAssertEqual(themes.count, 2)
+
+        let item = try XCTUnwrap(themes.first)
+        XCTAssertEqual(item.id, "tsubaki")
+        XCTAssertEqual(item.name, "Tsubaki")
+        // swiftlint:disable:next line_length
+        XCTAssertEqual(item.description, "Tsubaki")
```

Comment:
the description should be "Tsubaki puts the spotlight on your products and your customers. This theme leverages WooCommerce to provide you with intuitive product navigation and the patterns you need to master digital merchandising."

Reply:
Oh you already fixed it around the same time 👍🏼